src/Command/MeiliSearchCreateCommand.php

```diff
@@ -72,7 +72,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $output->writeln('<info>Creating index '.$index['name'].' for '.$entityClassName.'</info>');
 
-            $indexInstance = $this->searchClient->index($index['name']);
+            $task = $this->searchClient->createIndex($index['name']);
+            $this->searchClient->waitForTask($task['uid']);
+            $indexInstance = $this->searchClient->getIndex($index['name']);
```

Comment:
```suggestion
            $indexInstance = $this->searchClient->index($index['name']);
```

Reply:
If I'm correct, since the index is created, no need to get (using HTTP call), only an instance needs to be created.

Reply:
Good catch! I thought it could break by just creating the instance from a valid/real index in meili, but there is no need to be worried since no internal data is used to track that, so good catch, your suggestion was applied! ;)